src/presenters/admin/indexation-list-item-presenter.php

```diff
@@ -0,0 +1,59 @@
+<?php
+/**
+ * Presenter class for the indexation modal.
+ *
+ * @package Yoast\YoastSEO\Presenters\Admin
+ */
+
+namespace Yoast\WP\SEO\Presenters\Admin;
+
+use Yoast\WP\SEO\Presenters\Abstract_Presenter;
+
+/**
+ * Indexation_List_Item_Presenter class
+ */
+class Indexation_List_Item_Presenter extends Abstract_Presenter {
+
+	/**
+	 * The number of objects that need to be reindexed.
+	 *
+	 * @var int
+	 */
+	protected $total_unindexed;
+
+	/**
+	 * Indexation_List_Item_Presenter constructor.
+	 *
+	 * @param int $total_unindexed The number of objects that need to be indexed.
+	 */
+	public function __construct( $total_unindexed ) {
+		$this->total_unindexed = $total_unindexed;
+	}
+
+	/**
+	 * @inheritDoc
+	 */
+	public function present() {
+
+		$output = '<li><strong>' . esc_html__( 'Content indexation', 'wordpress-seo' ) . '</strong><br/>';
+
+		if ( $this->total_unindexed === 0 ) {
+			// phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped -- Reason: `message_already_indexed` is considered a safe method.
```

Comment:
Is this a leftover from before? This is the same output as in the integration again here.